src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -152,6 +152,7 @@ function mapStateToProps(state) {
   const facilityTypeName = facilityTypes[facilityType];
 
   return {
+    currentQuery: state.searchQuery,
```

Comment:
This was previously removed, but its removal was causing a runtime error so I'm adding it back.

Reply:
this might be used for keeping track of query state and pushing it to the URL